sync/class.jetpack-sync-functions.php

```diff
@@ -203,26 +210,26 @@ public static function get_protocol_normalized_url( $callable, $new_value ) {
 		$option_key = self::HTTPS_CHECK_OPTION_PREFIX . $callable;
 
 		$parsed_url = wp_parse_url( $new_value );
-		if ( ! $parsed_url ) {
+		if ( empty ( $parsed_url['scheme'] ) ) {
 			return $new_value;
 		}
 
-		$scheme = $parsed_url['scheme'];
-		$scheme_history = get_option( $option_key, array() );
+		$scheme           = $parsed_url['scheme'];
```

Comment:
This is where we need to see if the key exists, and I'd prefer to use `array_key_exists` here instead of `empty`.

Reply:
@zinigor 
If I use 
if ( array_key_exists ( 'scheme' , $parsed_url) ) {
  $scheme = $parsed_url['scheme'];
} else { 
 $scheme = '';
}
This is fine .